src/Cake.Common/Net/HttpAliases.cs

```diff
@@ -197,7 +197,7 @@ public static void DownloadFile(this ICakeContext context, Uri address, FilePath
             // We track the last posted value since the event seems to fire many times for the same value.
             var percentComplete = 0;
 
-            using (var http = new HttpClient())
+            using (var http = new HttpClient(new HttpClientHandler { UseDefaultCredentials = settings.UseDefaultCredentials }))
```

Comment:
If you like I can think of an API that would address this issue. But this would be definitely a breaking change. Your implementation is still irritating, because the user can still mix the different configuration.